ui/src/containers/SampleListViewContainer.jsx

```diff
@@ -308,6 +314,7 @@ class SampleListViewContainer extends React.Component {
     } else {
       this.props.syncSamples();
     }
+    this.sampleGridFilterByKey('limsSamples', true);
```

Comment:
Not sure the extra method is worth it. It would be clearer to write `this.props.filter({ limsSamples: true });`

Reply:
Ah, yes you are right a bit silly